libs/build_mapl.sh

```diff
@@ -22,6 +22,8 @@ if $MODULES; then
   module load esma_cmake
   module load cmakemodules
   module load ecbuild
+  # module exports ecbuild_ROOT, but when building without modules ECBUILD_ROOT is set
+  export ECBUILD_ROOT=$ecbuild_ROOT
```

Comment:
@aerorahul recommended using uppercase for environment variables set by modules. Should we just update the ecbuild module template instead?

Reply:
No. 
ecbuild is not a cmake package and it makes no difference either way. 
CMake uses the Package_ROOT variable in find_package(Package). 
We are not doing find_package(ecbuild)